docker/Dockerfile.tcf-dev

```diff
@@ -49,9 +49,14 @@ RUN apt-get update \
     cmake \
     protobuf-compiler \
     libprotobuf-dev \
-    python \
     python3-dev \
     virtualenv \
+    python3-pip \
+    python3-toml \
+    python3-requests \
+    python3-colorlog \
+    python3-twisted \
+    python3-protobuf \
```

Comment:
Are we using protobuf anywhere in our project? Remove it if we are not using it.

Reply:
I will remove it